services/supervisord/supervisord.go

```diff
@@ -486,6 +487,23 @@ func (s *Service) UpdateConfiguration(settings *models.Settings) error {
 // once we start generating it. See alertmanager service.
 
 var templates = template.Must(template.New("").Option("missingkey=error").Parse(`
+{{define "dbaas-controller"}}
+{{if .PerconaTestDbaas -}}
```

Comment:
I'm not sure that it's a good idea to do it this way, maybe it's better to pass value to autostart and autorestart values?

Reply:
btw, why these fields are missed?

Reply:
Sorry I didnt realize that it is mandatory. So fields added and value passed into these fields.